src/protocol.rs

```diff
@@ -16,204 +19,216 @@ const MAX_CID_SIZE: usize = 4 * 10 + 64;
 pub struct BitswapProtocol;
 
 impl ProtocolName for BitswapProtocol {
-    fn protocol_name(&self) -> &[u8] {
-        b"/ipfs-embed/bitswap/1.0.0"
-    }
+  fn protocol_name(&self) -> &[u8] {
+    b"/ipfs-embed/bitswap/1.0.0"
+  }
 }
```

Comment:
With the upcoming release, namely `v0.52.0` you can use `StreamProtocol` here. See https://github.com/libp2p/rust-libp2p/pull/3746/.